NA/us/california/irvinca-usa.k

```diff
@@ -1,9 +1,9 @@
 {
-    "68.96.80.118:46242":{
+    "fc3f:bc82:a104:8933:ba0f:2d2c:de82:966:64137":{
```

Comment:
because, It's more reliable. What if they don't have access to clearnet? Ipv4 is an overlay, however Ipv6 is avaliable even if you don't have an ISP.